projects/augeas/project.yaml

```diff
@@ -0,0 +1,8 @@
+homepage: https://github.com/hercules-team/augeas
+primary_contact: lutter@watzmann.net
```

Comment:
We will need one of the main project owners (as per the github project) to agree with you doing this.

Reply:
Hi, yes, I totally agree with this, and would love to have augeas added to oss-fuzz. Please let me know if this statement is enough or if I have to sign up somewhere else.

Reply:
@lutter Yes, this comment is sufficient, thanks!